docs/source/references/config.md

```diff
@@ -856,3 +813,29 @@ Has type `wasm::Configuration`. Can be configured via environment variable `WSV_
 }
 ```
 
+#### `wsv.wasm_runtime_config.fuel_limit`
+
+The fuel limit determines the maximum number of instructions that can be executed during the execution of a smart contract.
+
+Has type `Option<u64>`[^1]. Can be configured via environment variable `WASM_FUEL_LIMIT`
+
+```json
+1000000
+```
+
+#### `wsv.wasm_runtime_config.max_memory`
+
+Maximum amount of linear memory a given smartcontract can allocate
+
+Has type `Option<u32>`[^1]. Can be configured via environment variable `WASM_MAX_MEMORY`
+
+```json
+524288000
+```
+
+[^1]: Types wrapped in a single `Option<..>` signify that there is a fallback value for them specified in the corresponding `json` block, but it only serves as a reference. A default `null` for such types means that there is no meaningful fallback for this particular value available. The default peer `config.json` file can serve as a starting point, but there are only two final sources of truth for configuration: all the variables should either be provided via environment or in a file, and no other overloading happens. Failure to provide them in any of the above two ways results in an error.
```

Comment:
not sure I understood you correctly, so correct me if I am wrong, but I think this kind of explanation will be more clear to the users:

```suggestion
[^1]: A type wrapped in a single `Option<..>` signifies that in the corresponding `json` block there is a fallback value for this type, and that it only serves as a reference. The default `null` for such types means that there is no meaningful fallback for this particular value available. Note that all variables should either be provided via environment or in a file. Failure to provide them in any of the above two ways results in an error. This means that for configuration options that have types wrapped in a single `Option<..>` with `null` as the default, you *have to* to use environment variables.
```